pkg/apiserver/apiserver.go

```diff
@@ -38,8 +38,26 @@ import (
 
 	"github.com/emicklei/go-restful"
 	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
 )
 
+var (
+	apiserverLatencies = prometheus.NewSummaryVec(
+		prometheus.SummaryOpts{
+			Name: "apiserver_request_latencies",
+			Help: "Response latency summary in microseconds for each request handler, verb, and HTTP response code.",
+		},
+		[]string{"handler", "verb", "code"},
+	)
+)
+
+func init() {
+	// All Prometheus metrics have to be explicitly registered to appear on the metrics endpoint.
+	prometheus.MustRegister(apiserverLatencies)
+	prometheus.MustRegister(redirectCounter)
```

Comment:
nit: Init each of these in the file they're declared in. Or declare them here? We don't use apiserverLatencies here anyways.